### DISCUSSION TITLE
is liti alive or not?

### DISCUSSION THREAD

**Discussion by Blado** [gurkenlabs/litiengine#791] [General]
Hi, I'm trying java game engines right now. I only have one question: is this engine still alive or is it no longer being developed?

**Comment by steffen-wilke** [comment]
It's still being developed as you van see by the commot history. Currently working on the next update

**Reply by Blado** [reply]
Vielen Dank für die schnelle Antwort, war mir nur nicht sicher weil im Forum so still ist und die letzte Posts dort etwas negativen Hauch haben. Aber jetzt sehe ich dass da geantwortet wird, dass ich wenigstens eine Antwort bekomme falls ich irgendwelche Probleme habe.
